client/transport/src/ws/mod.rs

```diff
@@ -340,7 +340,8 @@ impl WsTransportClientBuilder {
 			tracing::debug!(target: LOG_TARGET, "Connecting to target: {:?}", target);
```

Comment:
The `sockaddrs` were being logged as part of `Target`, should we keep it in the logs ?

Reply:
It's quite noisy and the hostname is probably sufficient.

To debug such things i.e, that we may resolve the "wrong host" is harder but I'm fine omitting that.

Reply:
Sure, that makes sense.